repositories/level_role_repository.py

```
@@ -0,0 +1,92 @@
+"""LevelRoleRepository: Consolidated CRUD for level role assignments."""
+
+from collections.abc import AsyncIterator, Sequence
```

Flake8 (7.3.0)

[error] 'collections.abc.Sequence' imported but unused

(F401)